bakeup/templates/workshop/customerorder_form.html

```diff
@@ -2,20 +2,31 @@
 {% load i18n %}
 {% load django_bootstrap5 %}
 
-{% block title %}{% trans "Add new production day" %}{% endblock %}
+{% block title %}{% trans "Add neworder" %}{% endblock %}
 {% block page_heading %}
-{% if object %}{% trans "Update customer order" %}{% else %}{% trans "Add new production day" %}{% endif %}
+{% if object %}{% trans "Update customer order" %}{% else %}{% trans "Add order" %}{% endif %}
```

Comment:
New or not New isn't tgis the same AS above

Reply:
yes it is inconsistent all over the place :| Need to fix this at one point. Thanks for feedback!